cadence/src/client.rs

```diff
@@ -732,6 +734,13 @@ impl StatsdClientBuilder {
         self
     }
 
+    /// Add a default container ID to every metric published by the built
+    /// [StatsdClient].
+    pub fn with_container_id(mut self, container_id: &str) -> Self {
```

Comment:
Do you mind making `container_id` a `V: ToString` type similar to how the tag methods above work?

Reply:
done :)